perun-base/src/main/java/cz/metacentrum/perun/core/api/RichResource.java

```diff
@@ -130,6 +131,7 @@ public String toString() {
 
 		return str.append(getClass().getSimpleName()).append(":["
 			).append("id='").append(getId()
+			).append("', uuid='").append(getUuid()
```

Comment:
Setting of UUID is missing in constructor.

Reply:
Fixed.